src/main/java/qna/domain/BaseEntity.java

```diff
@@ -0,0 +1,32 @@
+package qna.domain;
+
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedBy;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@MappedSuperclass
+public abstract class BaseEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    protected Long id;
+
+    @CreatedDate
+    private LocalDateTime createAt;
+
+    @LastModifiedBy
+    private LocalDateTime updatedAt;
+
+    protected Long creatorId;
```

Comment:
writerId  를 creatorId 로 변경하신 이유가 있으신가요?
DB 컬럼명과 Entity 의 변수명을 다르게 설정하고 싶으실 경우 다음과 같이 해볼 수도 있어요 😄 
```suggestion
    @Column(name = "writer_id")
    protected Long creatorId;
```

Reply:
넵 DDL 그대로 작성하도록 하겠습니다!! 😄